src/DssSpell.t.base.sol

```diff
@@ -1030,56 +1030,77 @@ contract DssSpellTestBase is Config, DssTest {
         uint256 tinBps,
         uint256 toutBps
     ) internal {
-        uint256 tin = tinBps * WAD / 10000;
-        uint256 tout = toutBps * WAD / 10000;
+        uint256 tin  = tinBps  * WAD / 100_00;
+        uint256 tout = toutBps * WAD / 100_00;
         GemAbstract token = GemAbstract(join.gem());
 
+        // Check PIP is set (ilk exists)
         assertTrue(pip != address(0));
 
+        // Update price (poke spotter)
         spotter.poke(_ilk);
 
-        // Authorization
-        assertEq(join.wards(pauseProxy), 1);
-        assertEq(join.wards(address(psm)), 1);
-        assertEq(psm.wards(pauseProxy), 1);
-        assertEq(vat.wards(address(join)), 1);
-        assertEq(clip.wards(address(end)), 1);
+        // Authorization (check wards)
+        assertEq(join.wards(pauseProxy),    1);
+        assertEq(join.wards(address(psm)),  1);
+        assertEq(psm.wards(pauseProxy),     1);
+        assertEq(vat.wards(address(join)),  1);
+        assertEq(clip.wards(address(end)),  1);
 
-        // Check toll in/out
-        assertEq(psm.tin(), tin, _concat("Incorrect-tin-", _ilk));
+        // Check tin / tout values of PSM
+        assertEq(psm.tin(),  tin,  _concat("Incorrect-tin-",  _ilk));
         assertEq(psm.tout(), tout, _concat("Incorrect-tout-", _ilk));
 
-        // Increase ilk line to allow psm sell even if it is maxed out
+        // Arbitrary amount of TOKEN to test PSM sellGem and buyGem with (in whole units)
+        // `amount` is the amount of _TOKEN_ we are selling/buying (NOT measured in Dai)
+        uint256 amount = 100_000;
+        // Amount should be more than 10,000 as `tin` and `tout` are basis point measurements
+        require(amount >= 10_000, "checkPsmIlkIntegration/amount-too-low-for-precision-checks");
```

Comment:
If you are setting this in the line right above to a fixed value, this `require` is superfluous.

Reply:
Sorry, just saw this now, but this is to prevent future mistakes by crafters / updates.

Gas cost doesn't matter as this is a test, so I see no harm in this as there is a genuine reason to do so for coverage.